docs/src/api/class-locator.md

```diff
@@ -2478,6 +2478,18 @@ When all steps combined have not finished during the specified [`option: timeout
 ### option: Locator.uncheck.trial = %%-input-trial-%%
 * since: v1.14
 
+## method: Locator.visible
+* since: v1.51
+- returns: <[Locator]>
+
+Returns a locator that only matches [visible](../actionability.md#visible) elements.
+
+### option: Locator.visible.visible
+* since: v1.51
+- `visible` <[boolean]>
+
+Whether to match visible or invisible elements.
```

Comment:
Let's mention the default value here. Should this be an argument instead of an option? `visible(false)` looks better than `visible({ visible: false })`

Reply:
Interesting point. Let's discuss on the API review.